taichi/ui/common/scene_base.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include <vector>
+#include "taichi/ui/common/field_info.h"
+#include "taichi/ui/common/renderable_info.h"
+#include "taichi/ui/common/camera.h"
+#include "taichi/ui/utils/utils.h"
+
+TI_UI_NAMESPACE_BEGIN
+
+struct PointLight {
+  glm::vec3 pos;
+  glm::vec3 color;
+};
+
+struct MeshInfo {
+  RenderableInfo renderable_info;
+  glm::vec3 color;
+  float shininess;
```

Comment:
Maybe use something else here... If we really want to use PBR rendering then this is definitely not enough, if we don't use that we should just stick to lambertian on everything since this doesn't feel right here

Reply:
Ok good point, let's stick to lambertian for now.